base-android/src/main/java/com/sfaxdroid/base/utils/PixaUtils.kt

```diff
@@ -0,0 +1,8 @@
+package com.sfaxdroid.base.utils
+
+import androidx.lifecycle.ViewModel
+import javax.inject.Inject
+
+class PixaUtils
+    @Inject constructor(
```

Comment:
why inject

Reply:
I was going to inject something in here. I forgot what it was